src/schism/schism_nuopc_cap.F90

```diff
@@ -399,13 +404,17 @@ subroutine InitializeAdvertise(comp, importState, exportState, clock, rc)
     _SCHISM_LOG_AND_FINALIZE_ON_ERROR_(rc)
   enddo
 
+  ! for coupling to WW3
+  call NUOPC_FieldAdvertise(exportState, "ocn_current_zonal", "m s-1", localrc)
```

Comment:
For now, it's ok to use ocn_current_merid, but this is not a universal (or CF-compliant) name.  In the field creation, let's make sure we have CF information present.